.env.template

```diff
@@ -1,6 +1,6 @@
 # Copy this file and store it as ".env" and fill in the required environment variables.
 DATA_DIR=U:/Research/FloodRiskResearch/DigitalTwin/stored_data
-FLOOD_MODEL_DIR=U:/Research/FloodRiskResearch/DigitalTwin/BG-Flood/BG-Flood_Win10_v0.6-a
+FLOOD_MODEL_DIR=U:/Research/FloodRiskResearch/DigitalTwin/BG-Flood/BG-Flood-Model
```

Comment:
Could you put a file in here saying the version, or rename the folder back to the version number? Just something so that when we look in the BG-Flood folder we know what version it is

Reply:
renamed folder with version number